src/controllers/AuthController.js

```diff
@@ -0,0 +1,43 @@
+import jwt from 'jsonwebtoken';
+import helpers from '../helpers';
+import services from '../services';
+import findUser from '../services/findUser';
+
+const { forgotPasswordMessage } = helpers;
```

Comment:
instead of `import helpers from '../helpers'; ` and `const { forgotPasswordMessage } = helpers;`, can we just do something like `import { forgotPasswordMessage } from  from '../helpers'`

Reply:
Yeah, we can do that. The reason I did it this way is that we might be needing to import several helpers in the file. I felt it might not look so clean if we destructured them directly in the import statement. e.g `import { thisHelper, thatHelper, otherHelper, moreHelpers, evenMoreHelpers, andMoreHelpers } from '../helpers'`. I felt destructuring them on another line might be cleaner. What do you think?

Reply:
alright